.github/workflows/danger.yml

```diff
@@ -0,0 +1,14 @@
+name: Danger
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run Danger
+      uses: ./
```

Comment:
does this work?

Reply:
It sure does! 😸 I'll add the usage to the README

https://github.com/turekj/danger-py-ga-test/pull/1